File: ex1.c
#include <stdio.h>
#include <stdlib.h>

void conversor(float a, float b, float c, float d, float e, float f);

int main(void)
{
  float a, b, c, d, e, f;

  printf(" favor digitar os numeros na seguinte sequencia A B C D E F\n");
  scanf("%f %f %f %f %f %f", &a, &b, &c, &d, &e, &f);
  conversor(a,b,c,d,e,f);

  return 0;
}

void conversor(float a, float b, float c, float d, float e, float f)
{
  float x, y;

  y = ((f-d*c/a))/((-d*b)/(a+e));

  x = (c-b*y)/a;

  printf("o valor de y: %f\n", y);

  printf("o valor de x: %f\n", x);

  return;

}

File: python/ex1.py
print("=== ACHAR DUAS VARIAVEIS DE UM SISTEMA LINEAR ===\n")

#a, b, c, d, e, f = int(input("favor digitar os numeros A B C D E F na respectiva ordem usando uma linha\n").split())
a, b, c, d, e, f = [int(a) for a in input("Entre com os valores: ").split()]

y = ((f-d*c/a))/((-d*b)/(a+e))
x = ((c -b *y))/a

print("o valor de y é: ", y)
print("o valor de x é: ", x)
